lib/rspec/core/formatters/failure_list_formatter.rb

```diff
@@ -1,4 +1,6 @@
 RSpec::Support.require_rspec_core "formatters/base_formatter"
+require 'pathname'
```

Comment:
This can't be merged with any usage of `require 'pathname'` in lib, it's not always available and we don't want false positives for people (where they don't realise they needed to requite it).

Its also unnecessary, no other formatter uses it to look up failure locations. See the exception presenter etc for how we format lines.